Cargo.toml

```diff
@@ -56,6 +56,7 @@ members = [
     "tests-fuzz",
     "tests-integration",
     "tests/runner",
+    "src/dataflow",
```

Comment:
I think `flow` as the crate name is better. Because we call this module `GreptimeFlow`.

Reply:
Please keep the crate name in alphabetical order.